service/src/main/resources/db/migration/V355__application_document_refactoring.sql

```diff
@@ -0,0 +1,13 @@
+DROP VIEW IF EXISTS application_view;
+
+ALTER TABLE application
+    ADD COLUMN document     jsonb default '{}'                     NOT NULL,
+    ADD COLUMN form_modified timestamp with time zone default now() NOT NULL;
```

Comment:
Are you sure it's the best to have defaults and `NOT NULL` here? Wouldn't it be more "correct" have `document = NULL, form_modified = NULL` when the form has not been saved yet 🤔